src/utils/bolt11.ts

```diff
@@ -18,8 +18,16 @@ export const decodeBolt11 = (invoice: string) => {
    const amountSection = findSection(decoded.sections, 'amount');
    const amountSat = amountSection?.value ? Number(amountSection.value) / 1000 : undefined;
 
-   const expiryUnixSeconds = decoded.expiry;
-   const expiryUnixMs = decoded.expiry * 1000;
+   const expirySection = findSection(decoded.sections, 'expiry');
+   const timestampSection = findSection(decoded.sections, 'timestamp');
 
-   return { amountSat, expiryUnixMs };
+   /* infinity if no expiry is defined */
+   let expiryUnixSec = Infinity;
```

Comment:
should we maybe rather return undefined/null instead of inf?

Reply:
Do you think that's more clear? Seemed cleaner to me this way because then we can still treat it as a number.
Otherwise we have to do something like:
```ts
   const invoiceExpired = invoiceExpiry ? invoiceExpiry < now : false;
```

Reply:
hm yeah maybe you are right. I was thinking more from the ui perspective where if you somewhere want to show the inv expiry you'd do:
```
expiry ? <div>{expiry}</div> : null
```

instead of:
```
check not infinity ? <div>{expiry}</div> : null
```

Reply:
but I guess it's not hard to do `expiry != Infinity ? <div>{expiry}</div> : null` too so we can keep as is